src/hooks/useFhirClient.tsx

```diff
@@ -40,8 +44,15 @@ export function useFhirClient() {
     // TODO: add code, date, & other query param capabilities
     // TODO: consider using fhir-search across the board (documenting delay)
 
-    return useQuery(
-      [`search-${resourceType}`, params],
+    const fetchNextPage = () => {
+      if (hasMoreData) {
+        const pageSize = queryParams.pageSize || 10;
+        setNext((prev) => prev + pageSize);
```

Comment:
Is there a way we could use the url? I think the scroll api gives us a token instead of a page size.

Reply:
I'm not sure I fully understand. `patient-service` returns a url containing a `next` query param which is what dictates which set of elements to return. In the latest commit, `next` is exposed via the hook, and it needs to be passed to `fetchNext` to fetch the next set of data.